### ISSUE TITLE
[wg/webrtc] Web Real-Time Communications Working Group Charter

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/strategy#462]
New charter proposal, reviewers please take note.

# Charter Review

[Charter](https://w3c.github.io/webrtc-charter/webrtc-charter.html)

If applicable:
[diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2022%2F10%2Fwebrtc-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fwebrtc-charter%2Fwebrtc-charter.html)

The updated charter aligns it with the latest charter template changes and updates the list of deliverables.

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [*] Existing WG recharter 

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
https://github.com/w3c/webrtc-charter/issues

Anything else we should think about as we review? 
N/A

cc @aboba @alvestrand @jan-ivar

**Comment by simoneonofri** [comment]
Hi Dom,

For the Security part, I am reading the documentation, I would put as *must* the Security Considerations sections and in particular, always a reference to the general [Security Considerations from IETF](https://datatracker.ietf.org/doc/html/draft-ietf-rtcweb-security). Let me know if I can start a PR for that.

Where I would also consider the case of a Threat Actor who might develop an Agent that leverages this technology for its capabilities (e.g., https://beefproject.com).

It is also interesting to note that (after a quick test) it is possible to understand the capabilities of my end-user device even if I have not permitted to access the microphone, etc. (e.g., https://www.webrtc-experiment.com/DetectRTC/ code here
https://github.com/muaz-khan/DetectRTC).

On the [W3C Identity for WebRTC 1.0](https://www.w3.org/TR/webrtc-identity/) cc'ing @hlflanagan @wseltzer @asr-enid @timcappalli @marcoscaceres @samuelgoto

Thank you,

Simone

**Comment by dontcallmedom** [comment]
@simoneonofri - re using "MUST", my impression is that we've stayed away from using RFC2119 language in charters since they're not specifications; if we were to change that tack, I would want it to be discussed outside this particular rechartering, esp. since it is a mostly administrative rechartering of a group.

The IETF security considerations for WebRTC are linked from the WebRTC spec itself; they don't apply necessarily to all WebRTC Working Group specs though, so I'm not sure why or how they should be linked from the WG charter.

Issues with hardware detection in WebRTC (or more specifically, Media Capture and Streams) [have a long history](https://github.com/w3c/mediacapture-main/issues?q=is%3Aissue+enumerate+), and have seen recent spec improvements (not yet reflected in all implementations though).

If you do see potential changes to the charter, a PR would help for discussion indeed; thanks!

**Comment by himorin** [comment]
- Coordination section seems to have similar as charter-template, but a bit old one? I'd propose to update the latter half part with template text
- history table lacks links to several previous charters, and also several rows listed in past charters are missing (I'm not sure we should complete all here or not although)...

**Comment by dontcallmedom** [comment]
I've [aligned the coordination section with the template](https://github.com/w3c/webrtc-charter/commit/84ba492e14ff845dd23c1c885512d60b23a11462) and [completed the history section with links to previous charters](https://github.com/w3c/webrtc-charter/commit/836d4fe36ab07bc10e8e77bf6ff96799ffdba1cd)

**Comment by ruoxiran** [comment]
APA is happy with this charter.

**Comment by himorin** [comment]
no comment or request from i18n

**Comment by plehegar** [comment]
no comment or request from PING

**Comment by plehegar** [comment]
[Advance notice](https://lists.w3.org/Archives/Public/public-new-work/2024Jun/0002.html).

**Comment by simoneonofri** [comment]
[added the label from the past message]
